### PR TITLE
OUZ change in location code

### PR DESCRIPTION
**Pull request by mnaguit**
from LC 20 to LC 21

### CODE REVIEW

**Comment by mnaguit** [comment]
For your review @ozym

File: install/streams.csv [hunk 1]
```diff
@@ -1393,9 +1393,11 @@ OUZ,10,1,false,false,false,2010-02-24T00:00:01Z,9999-01-01T00:00:00Z
 OUZ,10,50,false,false,true,1991-03-12T03:26:00Z,2000-05-15T02:04:00Z
 OUZ,10,100,false,true,false,2002-08-30T08:00:00Z,2010-02-24T00:00:00Z
 OUZ,10,100,false,false,false,2010-02-24T00:00:01Z,9999-01-01T00:00:00Z
-OUZ,20,1,false,false,false,2003-08-26T05:00:00Z,9999-01-01T00:00:00Z
-OUZ,20,50,false,false,false,2010-02-24T00:00:01Z,9999-01-01T00:00:00Z
-OUZ,20,200,false,false,true,2003-08-26T05:00:00Z,9999-01-01T00:00:00Z
+OUZ,20,1,false,false,false,2003-08-26T05:00:00Z,2018-11-06T00:00:00Z
```
Comment (ozym) [inline install/streams.csv]: We can drop the 1 Hz for strong motion streams as this isn't in the archive.

Comment (ozym) [reply install/streams.csv]: (will be a wider cleanup step as well)

**Comment by ozym** [comment]
The issue with 1 Hz strong motion is a wide topic.

**Comment by mnaguit** [comment]
> The issue with 1 Hz strong motion is a wide topic.

Noted with thanks @ozym . As you have mentioned, this may entail a wider cleanup step. 
Will be useful to discuss it in our Delta meeting. FYI @samgeonet @elidana @salichon

**Pull request merged**